app/views/admin/help_requests/_custom_values.html.erb

```diff
@@ -5,6 +5,7 @@
     <div id="js-custom-fields-container" data-values='<%= @custom_values.to_json %>'>
       <custom-fields-panel 
         :request-path="'<%= custom_fields_admin_help_request_path(f.object.try(:id) || 0) %>'"
+        empty-point-error-message="<%= I18n.t("help_request.errors.empty_point") %>"
```

Comment:
hmm I guess we could follow [this soultion](https://www.stefanwienert.de/blog/2019/11/02/integrating-javascript-vue-i18n-into-rails-missing-auto-translate-pipeline/) to pass i18n into JS context... but not required